CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ This changelog track changes to the qoqo project starting at version v0.5.0
 ## 1.6.0-alpha.1
 
 * Renamed the PNRDetection operation to PhotonDetection.
+* Added support for JsonSchema.
```

Comment:
You'll need to put this in a 1.6.0-alpha.2 section and update to 1.6.0-alpha.2, as version 1.6.0-alpha.1 has already been released